.github/workflows/create-release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     # Current latest (0.14.0) has a runtime error
```

Comment:
Based on this comment it seems like you wanted to move back to the latest image after `0.14.0` was patched?